Gemfile

```diff
@@ -8,5 +8,7 @@ group :development do
   gem "test-unit", "~> 3.0", ">= 3.4.6"
   gem "test-unit-ruby-core"
   # In the case of Ruby whose rdoc is not a default gem.
-  gem "rdoc"
+  # Avoid installing rdoc gem in CI as a workaround of the following issue.
+  # https://github.com/ruby/openssl/issues/699
+  gem "rdoc" unless ENV["CI"]
```

Comment:
I think that `unless RUBY_ENGINE == "truffleruby"` is better than `unless ENV["CI"]`. Let me rebase the PR.

Reply:
I rebased the PR, updating the git commit message too.